test/fatptr/lit.cfg

```diff
@@ -0,0 +1,67 @@
+# -*- Python -*-
+
```

Comment:
check that you actually need a `lit.cfg`. Usually, the one from parent directory is inherited to all descendents. Only create a new configuration when existing ones do not work.

Reply:
AFAICT: '--horn-bv2-ptr-size=8', '--horn-bv2-fatmem' are two flags which are need for fatptr tests to pass but are not default for other tests. I will let this be for now since whether these flags work for every kind of test is orthogonal to this effort.